README.rst

```diff
@@ -531,7 +531,14 @@ with templates, this feature is also known as *configuration context*, think of
 it like a dictionary which is passed to the function which renders the
 configuration, so that it can fill variables according to the passed context.
 
-The different ways in which variables are defined are described below.
+The different ways in which variables are defined are described below in
+the order (high to low) of their precedence:
+
+1. `User defined device variables <#user-defined-device-variables>`_
+2. `Predefined device variables <#predefined-device-variables>`_
+3. `Global variables <#global-variables>`_
```

Comment:
Are you sure global variables would override group variables? I think it should be the opposite, group variables should override any global variable.